content/tutorial/02-sveltekit/01-concepts/04-server-side-rendering/README.md

```diff
@@ -2,4 +2,6 @@
 title: Server-side rendering
 ---
 
-TODO
+Server-side rendering (SSR) is the generation of HTML on the server. It's what SvelteKit does by default for the initial page load when a user visits your site.
+
+SSR tends to improve performance and makes your app accessible to users if JavaScript fails or is disabled (which happens [more often than you probably think](https://kryogenix.org/code/browser/everyonehasjs.html)). It is also preferred for search engine optimization (SEO) — while some search engines can index content that is rendered in the browser with JavaScript, it happens less frequently and reliably.
```

Comment:
I kinda wonder if we should fold this and the subsequent 'Prerendering' section into the previous 'Server and client' section, or at least combine them into a 'Rendering modes' section (that includes `csr`?).

Then again, all this is somewhat duplicative of content that will appear later, so I almost wonder if we should ruthlessly cut down the 'concepts' section.

Reply:
I was wondering the same above already - I'll take this as a "yes, remove it"